packages/repository-json-schema/src/__tests__/unit/filter-json-schema.unit.ts

```diff
@@ -172,15 +180,26 @@ describe('getFilterJsonSchemaFor', () => {
     ]);
   });
 
-  it('describes "order" as an array', () => {
-    const filter = {order: 'invalid-order'};
+  it('describes "order" as a string or array', () => {
+    const filter = {order: {invalidOrder: ''}};
     ajv.validate(customerFilterSchema, filter);
     expect(ajv.errors ?? []).to.containDeep([
+      {
+        keyword: 'type',
+        dataPath: '.order',
+        message: 'should be string',
+      },
       {
         keyword: 'type',
         dataPath: '.order',
         message: 'should be array',
       },
+      {
+        keyword: 'oneOf',
+        dataPath: '.order',
+        params: {passingSchemas: null},
+        message: 'should match exactly one schema in oneOf',
+      },
```

Comment:
Hmm, if I am reading this correctly, the error message is not very descriptive. As I understand this data, it says:

```
.order should be string
.order should be array
.order should match exactly on schema in oneOf
```

I find that a bit confusing, but I guess it's a problem in AJV, not something we can fix at LoopBack side. Would you mind to open an issue in [AJV bug tracker](https://github.com/ajv-validator/ajv/issues)? I did a quick search and did not find any relevant issue.